### ISSUE TITLE
New project creation fails (scripted and manual)

### ISSUE THREAD

**Issue by treedy** [andersfylling/disgord#115]
**Describe the bug**
The `disgord.sh` script and the manual install process steps fail at the build stage.

**Expected behavior**
I would expect that the initial project builds and that it can be run.

**Error messages**
_Output from disgord.sh_
```
# # # # # # # # # # # # # # # # #
#        DisGord utility        #
#            v0.0.0             #
# # # # # # # # # # # # # # # # #
* Simple script to assist you   *
* with creating a basic DisGord *
* bot.                          *
* * * * * * * * * * * * * * * * *

project name (mybot):
bot prefix (!): !
project path (.): .
git support (y/n): n
Creating project
go: creating new go.mod: module mybot
# mybot
./main.go:21:9: unknown field 'BotToken' in struct literal of type disgord.Config
./main.go:22:9: unknown field 'Logger' in struct literal of type disgord.Config
./main.go:27:22: undefined: disgord.NewClient
./main.go:41:15: botConfig.Logger undefined (type *disgord.Config has no field or method Logger)
```
---
_Output from manual process_
```
 % set | grep GO
GO1115MODULE=auto
GO111MODULE=auto
GOBIN=/home/treedy/.gvm/pkgsets/go1.11.5/global/bin
GOPATH=/home/treedy/.gvm/pkgsets/go1.11.5/global
GOROOT=/home/treedy/.gvm/gos/go1.11.5

treedy@devshell [15:51:34] [~/mounted_home]
-> % mkdir my-bot && cd my-bot # Not in GOPATH

###
# Edited main.go to have contents from Step 2
###

treedy@devshell [15:58:44] [~/mounted_home/my-bot]
-> % export GO111MODULE=auto
treedy@devshell [15:58:48] [~/mounted_home/my-bot]
-> % go mod init my-bot
go: creating new go.mod: module my-bot
treedy@devshell [15:58:57] [~/mounted_home/my-bot]
-> % go build .
# my-bot
./main.go:7:12: undefined: disgord.New
./main.go:8:3: unknown field 'BotToken' in struct literal of type disgord.Config
./main.go:9:3: unknown field 'Logger' in struct literal of type disgord.Config

```

**Desktop (please complete the following information):**
 - Golang version: 1.11.5
 - Using Go modules? yes

**Additional context**
```
-> % uname -a
Linux devshell 4.19.4-02480-gd44d301822f0 #1 SMP PREEMPT Thu Dec 6 17:45:27 PST 2018 x86_64 x86_64 x86_64 GNU/Linux
treedy@devshell [15:51:07] [~/mounted_home/mybot]
-> % go version
go version go1.11.5 linux/amd64
```

**Comment by andersfylling** [comment]
Hi, this is just go modules picking DisGord v0.8.8 instead of the beta version of v0.9. However, I'll make some changes to the script now and release v0.9 asap as it seems stable anyways. I'll close this issue once the latest version is out (with a fix).

Thanks for you bug report.

**Comment by andersfylling** [comment]
Hi, this issue should now be resolved in the latest version v0.9.1.

Please remember to use the latest disgord.sh script if you have it locally.
